README.rst

```diff
@@ -56,11 +56,11 @@ Listed below are the actively maintained release branches along with their lates
 patch release, corresponding image pull tags and their release notes:
 
 +---------------------------------------------------------+------------+------------------------------------+----------------------------------------------------------------------------+
-| `v1.16 <https://github.com/cilium/cilium/tree/v1.16>`__ | 2024-09-20 | ``quay.io/cilium/cilium:v1.16.2``  | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.16.2>`__  |
+| `v1.16 <https://github.com/cilium/cilium/tree/v1.16>`__ | 2024-10-10 | ``quay.io/cilium/cilium:v1.16.3``  | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.16.3>`__  |
 +---------------------------------------------------------+------------+------------------------------------+----------------------------------------------------------------------------+
-| `v1.15 <https://github.com/cilium/cilium/tree/v1.15>`__ | 2024-09-20 | ``quay.io/cilium/cilium:v1.15.9``  | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.15.9>`__  |
+| `v1.15 <https://github.com/cilium/cilium/tree/v1.15>`__ | 2024-10-10 | ``quay.io/cilium/cilium:v1.15.10``  | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.15.10>`__  |
```

Comment:
i guess table columns need to be aligned https://github.com/cilium/cilium/actions/runs/11365393856/job/31613497121?pr=35412 👀

Reply:
fixed :pray: